.styleguide

```diff
@@ -13,6 +13,8 @@ generatedFileExclude {
   ni-libraries/include/
   ni-libraries/lib/
   FRCNetComm\.java$
+  simulation/frc_gazebo_plugins/frcgazebo
```

Comment:
This should have / appended so it matches just a directory named "frcgazebo" and nothing else.